File: src/main/java/com/lcaohoanq/models/UserGoogle.java
package com.lcaohoanq.models;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.Getter;
import lombok.ToString;

@Getter
@ToString
@AllArgsConstructor
public class UserGoogle {

	private String id;
	private String email;
	private boolean verified_email;
	private String name;
	private String given_name;
	private String family_name;
	private String picture;

}

File: src/main/java/com/lcaohoanq/models/UserGoogle.kt
package com.lcaohoanq.models

data class UserGoogle(
    val id: String? = null,
    val email: String? = null,
    val verified_email: Boolean = false,
    val name: String? = null,
    val given_name: String? = null,
    val family_name: String? = null,
    val picture: String? = null
)

fun main(args: Array<String>) {
    val userGoogle = UserGoogle(
        name = "Hoang",
        given_name = "Hoan",
        family_name = "Nguyen",
        email = "hoang",
        picture = "https://www.google.com",
        verified_email = true
    )
    println(userGoogle)
}
